source/lib/config/user_data/data_user_data

```diff
@@ -59,6 +59,7 @@ user=${USER_NAME}
 autorestart=true
 redirect_stderr=true
 stdout_logfile=/var/log/supervisor/historical.log
+stopwaitsecs=30
```

Comment:
We repeat this for each node type, maybe we can put this in `common user_data` like?